openmetadata-ui/src/main/resources/ui/src/components/Persona/PersonaSelectableList/PersonaSelectableList.test.tsx

```diff
@@ -0,0 +1,172 @@
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { act, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { PersonaSelectableList } from './PersonaSelectableList.component';
+
+const data = [{ id: '1', type: 'user', name: 'username' }];
+
+const allData = [
+  { id: '1', type: 'user', name: 'username' },
+  { id: '2', type: 'user', name: 'username1' },
+];
+
+jest.mock('../../../rest/PersonaAPI', () => ({
+  getAllPersonas: jest.fn().mockResolvedValue({
+    data: allData,
+    paging: { total: 0 },
+  }),
+}));
+
+jest.mock('../../../utils/EntityUtils', () => ({
+  getEntityName: jest.fn().mockReturnValue('username'),
+  getEntityReferenceListFromEntities: jest.fn().mockReturnValue(allData),
+}));
+
+jest.mock('react-i18next', () => ({
+  useTranslation: jest.fn().mockReturnValue({
+    t: (label: string) => label,
+  }),
+}));
+
+jest.mock('../../common/SelectableList/SelectableList.component', () => ({
+  SelectableList: jest
+    .fn()
+    .mockImplementation(
+      ({
+        children,
+        customTagRenderer,
+        searchPlaceholder,
+        fetchOptions,
+        onUpdate,
+      }) => (
+        <div data-testid="selectable-list">
+          {children}
+          {searchPlaceholder}
+          {customTagRenderer('add-user')}
+          <button
+            data-testid="fetch-options"
+            onClick={() => fetchOptions('username')}>
+            fetchOptions
+          </button>
+          <button data-testid="update" onClick={onUpdate}>
+            Update
+          </button>
+        </div>
+      )
+    ),
+}));
+
+jest.mock('antd', () => ({
+  ...jest.requireActual('antd'),
+
+  Popover: jest.fn().mockImplementation(({ content, children, showArrow }) => {
+    return (
+      <div data-testid="popover">
+        {content}
+        {children}
+        <button data-testid="show-arrow-button" onClick={() => showArrow(true)}>
+          showArrow
+        </button>
+      </div>
+    );
+  }),
+}));
+
+const mockUpdate = jest.fn();
+
+describe('PersonaSelectableList', () => {
+  it('should render Persona Selectable List', () => {
+    render(
+      <PersonaSelectableList
+        hasPermission
+        multiSelect={false}
+        selectedPersonas={data}
+        onUpdate={mockUpdate}
+      />
+    );
+
+    expect(screen.getByTestId('popover')).toBeInTheDocument();
+    expect(screen.getByTestId('selectable-list')).toBeInTheDocument();
+    expect(screen.getByTestId('add-user')).toBeInTheDocument();
+  });
+
+  it('should show the persona list', async () => {
+    await act(async () => {
+      render(
+        <PersonaSelectableList
+          hasPermission
+          multiSelect={false}
+          selectedPersonas={data}
+          onUpdate={mockUpdate}
+        />
+      );
+    });
+
+    expect(screen.getByTestId('persona-list-item')).toBeInTheDocument();
```

Comment:
Can we check for persona name instead of element?

Reply:
yes it is being checked on the next line